### ISSUE TITLE
adddilation-stridewith8

### ISSUE THREAD

**Issue by shiyanrubing** [meteorshowers/RCF-pytorch#3]
您好，我最近在学习您的RCF-pytorch 代码， 遇到了一点问题：adddilation-stridewith8 是什么意思？ 是要将涉及到dilation卷积的步长调整到8？  比如， 在您的 models.py 文件中定义的类 class FlipConv2d(_ConvNd):
    def __init__(self, in_channels, out_channels, kernel_size, stride=1,
                 padding=0, dilation=1, groups=1, bias=True):
是要将 stride=1 调整为 8。 不知我的理解是否正确？ 提前多谢您的指导哈，祝您有一个愉快的中秋。

**Comment by meteorshowers** [comment]
sorry for the late response~
this code is not for rcf model

**Comment by shiyanrubing** [comment]
where or which are the parameters of RCF model?

**Comment by meteorshowers** [comment]
just load vgg model is ok

**Comment by shiyanrubing** [comment]
I mean that the parameters are not prepared model parameters, and they are the ones like adam initial learning rating, weight decay and so on.

**Comment by shiyanrubing** [comment]
In your "Method",  such as adam1e-4、 sgd1e-6-tunelr-caffecrop-fixbalanceweightto1.1,  not for RCF model?

**Comment by meteorshowers** [comment]
because pytorch is different from caffe, so, I change the optimizer to adam.
the caffe version use the sgd optimizer.
